concurrency/WFLY-12896_EE_Concurrency_Hung_Tasks_Termination.adoc

```diff
@@ -0,0 +1,97 @@
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12896[WFLY-12896]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1397[EAP7-1397]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:tborgato@redhat.com[Tommaso Borgato]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* WildFly
+
+=== Other Interested Projects
+
+* NONE
+
+== Requirements
+
+=== Hard Requirements
+
+* Add the following attribute to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-executor-service/[/subsystem=ee/managed-executor-service resource]:
+** *hung-task-termination-period*: the period, in milliseconds, for hung tasks automatic termination. A value of 0, which is the default, deactivates the feature.
+
+CLI to read the attribute from the Managed Executor Service resource named *default*:
+----
+/subsystem=ee/managed-executor-service=default:read-attribute(name=hung-task-termination-period)
+----
+
+* Add the following operation to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-executor-service/[/subsystem=ee/managed-executor-service resource]:
+** *terminate-hung-tasks*: manually terminates all executor tasks that are hung.
+
+CLI to execute the operation from the Managed Executor Service resource named *default*:
+----
+/subsystem=ee/managed-executor-service=default:terminate-hung-tasks()
+----
+
+* Add the following attribute to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-scheduled-executor-service/[/subsystem=ee/managed-scheduled-executor-service resource]:
+** *hung-task-termination-period*: the period, in milliseconds, for hung tasks automatic termination. A value of 0, which is the default, deactivates the feature.
+
+CLI to read the attribute from the Managed Scheduled Executor Service resource named *default*:
+----
+/subsystem=ee/managed-scheduled-executor-service=default:read-attribute(name=hung-task-termination-period)
+----
+
+* Add the following operation to the https://wildscribe.github.io/WildFly/18.0/subsystem/ee/managed-scheduled-executor-service/[/subsystem=ee/managed-scheduled-executor-service resource]:
+** *terminate-hung-tasks()*: manually terminates all executor tasks that are hung.
+
+CLI to execute the operation from the Managed Scheduled Executor Service resource named *default*:
+----
+/subsystem=ee/managed-scheduled-executor-service=default:terminate-hung-tasks()
+----
+
+=== Nice-to-Have Requirements
+
+* NONE
+
+=== Non-Requirements
+
+* Domain mode: the new attributes discussed in this proposal will be provided on the servers in a managed domain, but will not be registered with the /profile=*/subsystem=ee resources.
```

Comment:
@emmartins I might have overlooked this statement ... the meaning it's not clear to me: what does it mean that the new attributes will be provided but not registered? what are the implications? which is the reason why domain mode is treated differently?

Reply:
This was content from another AD which removal I missed, there is nothing particular/different wrt domain mode on this RFE